Code/Framework/AzCore/AzCore/std/tuple.h

```diff
@@ -345,5 +345,5 @@ namespace std
 // Adds typeinfo specialization for tuple type
 namespace AZ
 {
-    AZ_TYPE_INFO_INTERNAL_SPECIALIZED_TEMPLATE_PREFIX_UUID(AZStd::tuple, "tuple", "{F99F9308-DC3E-4384-9341-89CBF1ABD51E}", AZ_TYPE_INFO_INTERNAL_TYPENAME_VARARGS);
+    AZ_TYPE_INFO_INTERNAL_SPECIALIZED_TEMPLATE_PREFIX_UUID(AZStd::tuple, "AZStd::tuple", "{F99F9308-DC3E-4384-9341-89CBF1ABD51E}", AZ_TYPE_INFO_INTERNAL_TYPENAME_VARARGS);
```

Comment:
With this change, wouldn't you also need to change the Python unit tests here?
https://github.com/o3de/o3de/blob/15e4bbbb5c177e522ab8a3be5c4710686f7d2c16/Gems/EditorPythonBindings/Code/Tests/PythonTupleTests.cpp#L240-L241

For example the line:
```
test_tuple = azlmbr.object.create('tuple<bool int float >')
```
Should need to be changed to:
```
test_tuple = azlmbr.object.create('AZStd::tuple<bool, int, float>')
```
(i.e. "tuple" -> "AZStd::tuple", commas between the params, no extra space after the last param)

Reply:
Yes, that needs to be updated.
I'll update that now